util/report_utils/extract_csv.rb

```diff
@@ -0,0 +1,62 @@
+# TODO: move to perf_results_helper.rb
+# TODO: update to call csv2html
+
+require 'json'
+require 'csv'
+
+raise Exception, 'you must provide a results directory' unless ARGV[0]
+results_dir = ARGV[0]
+
+# TODO: easier way using file_utils?
+results_name = results_dir.split('/')[results_dir.split('/').length - 1]
+stats_path = ARGV[0] + '/js/stats.json'
```

Comment:
You have already given `ARGV[0]` the variable name `results_dir`.  Please use that variable name here.

Reply:
Thanks; updated